seed/challenges/html5-and-css.json

```diff
@@ -3406,8 +3406,8 @@
         "<code>}</code>"
       ],
       "tests": [
-        "assert($(\"body\").css(\"background-color\") === \"rgb(0, 0, 0)\", 'Give your <code>body</code> element the <code>background-color</code> of black.')",
-        "assert(editor.match(/<style>\\s*body\\s*\\{\\s*background-color\\s*:\\s*black;?\\s*\\}\\s*<\\/style>/i), 'Make sure your CSS rule is properly formatted with both and opening and closing curly bracket.')"
+        "assert($(\"body\").css(\"background-color\") === \"rgb(0, 0, 0)\" && editor.match(/background-color/g), 'Give your <code>body</code> element the <code>background-color</code> of black.')",
```

Comment:
I think this will work (can't test right now) not the problem is that your test is looking for `background-color` in editor and checks if `body`'s `background-color` is black at the same time but this is not what we're expecting reading the case message.
Please, resolve merge conflicts, and move this to next `assert`: replace `background.*` with `background-color` and leave first case unchanged, thanks a lot.